docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    env_file:
+      - .env
+    depends_on:
+      - db
+    volumes:
+      - ./:/app
+
+  db:
+    image: postgres:14-alpine
+    ports:
+      - "5433:5432"
```

Comment:
The database port mapping exposes the PostgreSQL port to the host machine. Ensure this is necessary for your development or testing purposes. In production, it's generally safer to avoid exposing database ports unless required.